### PR TITLE
Update Android part of flutter_view to use the new platform message c…

### PR DESCRIPTION
**Pull request by szakarias**
…hannel.

### CODE REVIEW

File: examples/flutter_view/android/app/src/main/java/com/example/view/MainActivity.java [hunk 1]
```diff
@@ -51,11 +57,15 @@ protected void onCreate(Bundle savedInstanceState) {
...
+        messageChannel =
+            new FlutterMessageChannel<String>(flutterView, CHANNEL, StringMessageCodec.INSTANCE);
+        messageChannel.
+            setMessageHandler(new MessageHandler<String>() {
                 @Override
-                public String onMessage(FlutterView view, String message) {
-                    return onFlutterIncrement();
+                public void onMessage(String s, Reply<String> reply) {
+                    onFlutterIncrement();
+                    reply.send(EMPTY_MESSAGE );
```
Comment (abarth) [inline examples/flutter_view/android/app/src/main/java/com/example/view/MainActivity.java]: Looks like you've got an extra space after EMPTY_MESSAGE.

**Comment by abarth** [comment]
LGTM but not to Travis.

**Comment by abarth** [comment]
```
>>> RUNNING in .: bin/flutter analyze --flutter-repo
Analyzing 967 files...
[lint] Specify type annotations. (examples/flutter_view/lib/main.dart, line 37, col 13)
```

File: examples/flutter_view/lib/main.dart [hunk 2]
```diff
@@ -47,13 +50,13 @@ class _MyHomePageState extends State<MyHomePage> {
...
   void _sendFlutterIncrement() {
-    PlatformMessages.sendString(_channel, _emptyMessage);
+    platform.send(_pong);
   }
 
   @override
   Widget build(BuildContext context) {
-    PlatformMessages.setStringMessageHandler(_channel,
-                                             handlePlatformIncrement);
+    platform.setMessageHandler(handlePlatformIncrement);
```
Comment (mravn-google) [inline examples/flutter_view/lib/main.dart]: This can be moved to `initState`

File: examples/flutter_view/lib/main.dart [hunk 3]
```diff
@@ -47,13 +50,13 @@ class _MyHomePageState extends State<MyHomePage> {
   }
```
Comment (mravn-google) [inline examples/flutter_view/lib/main.dart]: You may consider making `handlePlatformIncrement` private. And perhaps merge it with `_incrementCounter` to keep the number of methods small.

**Review by mravn-google — APPROVED** [review]
LGTM

**Comment by szakarias** [comment]
Thanks for the comments, all addressed.

**Pull request merged**